server/controllers/communityControllers/community.controller.js

```diff
@@ -0,0 +1,53 @@
+const knex = require('../../utils/knex').instance();
+
+module.exports.createOneCommunityUser = async (req, res) => { // Creates Community Info for one user
+  knex('users').select('user_id').where({ email: req.user.email }).then((id) => {
+    knex('community_tokens').where({ user_id: id[0].user_id }).then(user => {
+      if (user.length) { // User Exists Already
+        res.status(500).json({ message: 'User Already Exists!', err: 'Community User Exists' });
```

Comment:
Not a big deal, but I assume we should set the status code as 400 instead of 500  in the case when the user already exists since it's on client side error?